libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorAdaptiveRecognizer.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Schema;
+using Microsoft.Orchestrator;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.AI.Orchestrator
+{
+    public class OrchestratorAdaptiveRecognizer : Recognizer
+    {
+        [JsonProperty("$kind")]
+        public const string Kind = "Microsoft.OrchestratorRecognizer";
+
+        /// <summary>
+        /// Intent name that will be produced by this recognizer if the child recognizers do not have consensus for intents.
+        /// </summary>
+        private const string ChooseIntent = "ChooseIntent";
+
+        /// <summary>
+        /// Property name for candidate intents that meet the ambiguity threshold.
+        /// </summary>
+        private const string CandidatesCollection = "candidates";
+
+        private static string modelPath = null;
```

Comment:
These null initializations are not needed.

Reply:
Fixed.,